features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -859,6 +859,15 @@
         <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.windcentrale/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-xmppclient" description="XMPP Client Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <bundle dependency="true">wrap:mvn:org.igniterealtime.smack/smack-java7/4.3.1</bundle>
```

Comment:
Please add some more information (bundle-name etc.). Have a look at other features using `wrap`. Otherwise the resolves bundles look bad in karaf console.

Reply:
I'm not sure yet if wrapping is necessary. That was a try to check if travis passes last time, but @pavel-gololobov screwed the branch up. If travis no passes, the mentioned information must be added of course